CONTRIBUTING.md

```diff
@@ -214,3 +214,49 @@ All work on GOV.UK should [use progressive enhancement](https://www.gov.uk/servi
 Services on GOV.UK are for the benefit of all users. This means you must build a service that’s as inclusive as possible.
 
 [Accessibility guidance on GOV.UK](https://www.gov.uk/service-manual/helping-people-to-use-your-service/making-your-service-accessible-an-introduction).
+
+### Code Style (Linting)
+
+We use tools to ensure the style of our SCSS and JS is consistent.  The code is checked when the tests
+run on the build server.  If you are contributing SCSS or JS you'll probably be running
+the tests using `npm test` - this includes the code style checks.
+
+To run the code style checks independently you can run `npm run lint`.
+
+Automatic fixing is available for a number of the code style rules we apply.  To let the tool
+automatically fix the issues it can fix run `npm run lint:fix` - that will show you any remaining
+issues after the tool has fixed what it can for you.
+
+### WebJar publishing
+
+In order to make the hmrc-frontend assets easy to consume in a Play framework-based environment,
+the npm `build:webjar` task builds a Play-compatible webjar. This webjar is published to HMRC's open
```

Comment:
Maybe Scala or sbt compatible, rather than Play compatible.

Reply:
Updated to say JVM compatible, as webjars aren't specific to Scala/Play.